qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -80,30 +89,31 @@ class DefaultPoliciesTest extends BaseSpecification {
 
     static final private Deployment STRUTS_DEPLOYMENT = new Deployment()
             .setName(STRUTS)
-            .setImage("quay.io/rhacs-eng/qa:struts-app")
+            .setImage(STRUTS_IMAGE)
             .addLabel("app", "test")
             .addPort(80)
 
     static final private List<Deployment> DEPLOYMENTS = [
         new Deployment()
             .setName (NGINX_LATEST)
-            .setImage ("quay.io/rhacs-eng/qa:latest") // this is docker.io/nginx:1.22-alpine but tagged as latest
+            // this is docker.io/nginx:1.23.3 but tagged as latest
+            .setImage ("quay.io/rhacs-eng/qa-multi-arch-nginx:latest")
             .addPort (22)
             .addLabel ("app", "test")
             .setEnv([SECRET: 'true']),
         STRUTS_DEPLOYMENT,
+        // new Deployment()
```

Comment:
Is this needed?

Reply:
`SSL_TERMINATOR` deployment is not needed since the test was originally disabled out @ https://github.com/stackrox/stackrox/blob/e6cf8b2f7a406721711a2f3ebfdefe28f59a598b/qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy#L267
Also we don't have multi-arch image for `quay.io/rhacs-eng/qa:ssl-terminator`. so we just commented the deployment code to avoid pod carshloop error on P/Z.

Reply:
Makes sense. Originally removed here: https://github.com/stackrox/rox/pull/1555/files